backend/app/api/routes/documents.py

```
@@ -38,24 +47,80 @@
     description=load_description("documents/upload.md"),
     response_model=APIResponse[Document],
 )
-def upload_doc(
+async def upload_doc(
     session: SessionDep,
     current_user: CurrentUser,
     src: UploadFile = File(...),
+    background_tasks: BackgroundTasks = None,
+    target_format: Optional[str] = Form(None),
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)